torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -41,13 +41,21 @@ def is_fixed_qparams_node(node, modules):
     return is_call_function, is_call_method, is_call_module
 
 # Mapping from reference module class to the replacement quantized module class for lowering
-LOWER_MODULE_MAP: Dict[Type[nn.Module], Type[ReferenceableQuantizedModule]] = {
+# TODO: fix typing, the key is reference module
+LOWER_MODULE_MAP: Dict[Type[torch.nn.Module], Type[ReferenceableQuantizedModule]] = {
     nnqr.Linear: nnq.Linear,
     nnqr.Conv1d: nnq.Conv1d,
     nnqr.Conv2d: nnq.Conv2d,
     nnqr.Conv3d: nnq.Conv3d,
 }
 
+# TODO: merge with LOWER_MODULE_MAP after we merge
+# _lower_weighted_ref_module and special_pattern_replacement
+LOWER_MODULE_MAP2 = {
```

Comment:
maybe name it `LOWER_MODULE_MAP_TEMP`?

Reply:
or maybe `SPECIAL_PATTERN_LOWER_MODULE_MAP` if we're using it for `special_pattern_replacement` only